charms/candid/tests/integration/test_relations.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import logging
+
+import pytest
+from pytest_operator.plugin import OpsTest
+
+import integration.utils as utils
+
+logger = logging.getLogger(__name__)
+
+APP_NAME = "candid"
```

Comment:
I think these probably belong in conftest under config, see for example: https://stackoverflow.com/questions/71131298/how-can-i-see-the-current-pytest-configuration

Reply:
isn't this an unnecessary complication? for such a simple thing as defining a few strings?